athf/core/metrics.py

```
@@ -168,15 +168,20 @@ def append(self, event: MetricEvent) -> None:
                     fh.write(line)
 
     def _append_flock(self, line: str) -> None:
-        import fcntl
+        import fcntl  # POSIX-only; guarded by _has_fcntl() at the call site
 
+        # Access attributes via getattr so mypy on Windows (where the fcntl
+        # stub is empty) does not flag them; the call site guards on _has_fcntl.
+        flock = getattr(fcntl, "flock")
```

Ruff (0.15.21)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)